schemas/io.snowplow.ecomm/return_search/jsonschema/1-0-0

```diff
@@ -0,0 +1,28 @@
+{
+    "description": "When the submit_search is completed, this event will be triggered along with the contexts view_collection (the viewing of the search results) and search (the search event parameters).",
```

Comment:
Tempted to name it `search_result`.

Reply:
Since we have submit_search probably best to leave that naming convention as is. Also the name `search_result` denotes that the search returned with some result but it is possible to have searches that have no results.